### DISCUSSION TITLE
Why here is `error.gif` instead of `Client.imageBasePath + 'error.gif'`

### DISCUSSION THREAD

**Discussion by ParadiseWitch** [maxGraph/maxGraph#238] [Q&A] [answered]
https://github.com/maxGraph/maxGraph/blob/e25fd447be9ff3c02b877928ed17a615a4d77896/packages/core/src/util/Utils.ts#L51

**Comment by tbouffard** [comment, marked as answer]
Hi @ParadiseWitch 
This is probably a mistake introduced during the migration from _mxGraph_ or something that has changed to make a storybook example work.

The _mxGraph_ original code used what is currently commented: https://github.com/jgraph/mxgraph/blob/ff141aab158417bd866e2dfebd06c61d40773cd2/javascript/src/js/util/mxUtils.js#L41-L45

In the other places where images are used, `Client.imageBasePath` is involved:
- https://github.com/maxGraph/maxGraph/blob/e25fd447be9ff3c02b877928ed17a615a4d77896/packages/core/src/gui/MaxWindow.ts#L315-L338
- https://github.com/maxGraph/maxGraph/blob/e25fd447be9ff3c02b877928ed17a615a4d77896/packages/core/src/gui/MaxPopupMenu.ts#L89

In the past, fixes related to such kinds of images were done:
- https://github.com/maxGraph/maxGraph/pull/77#pullrequestreview-861815060
- https://github.com/maxGraph/maxGraph/pull/87

So, this seems a bug.

@ParadiseWitch would you be interesting in providing a Pull Request? It would be nice to have an example of usage of this image in Storybook (it may already exist 😄)

**Reply by tbouffard** [reply]
I didn't get a reply, so I solved the problem myself.
See #241 which will be included in version `0.4.0`
